packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -130,6 +131,13 @@ class Dart2JSTarget extends Target {
       ? PackageMap.globalGeneratedPackagesPath
       : PackageMap.globalPackagesPath;
     final File outputFile = environment.buildDir.childFile('main.dart.js');
+
+    // Dart defines are encoded as a JSON array, so we need to parse it. Also JSON
+    // parser returns List<dynamic>, so we need to cast it.
```

Comment:
You don't actually need to cast it, since you are only calling toString treat it as a List<Object>

Reply:
Edited: github hit my `<Object>`

Reply:
Done.